src/map.rs

```diff
@@ -257,6 +267,84 @@ impl Map {
     }
 }
 
+/// Represents the way tiles are laid out in a map.
+#[derive(Debug, PartialEq, Eq, Copy, Clone, Default)]
+#[allow(missing_docs)]
+pub enum StaggerIndex {
+    #[default]
```

Comment:
Not that it matters a lot because the property is always written out for staggered or hexagonal maps, but in Tiled the default is `Odd`.